### PR TITLE
[UPDATE BONUS GUIDE] JoinMarket webui (Jam) v0.1.5

### PR DESCRIPTION
**Pull request by dutu**
#### What

1. Update Bonus guide: JoinMarket webui (Jam) to v0.1.5.
2.  Fix Tor port (change from 80 to 443).

### Why

1. New [Jam release version v0.1.5](https://github.com/joinmarket-webui/jam/releases).
2. Jam is listening on SSL port (4020), hence Tor should listen on and forward port 443 rather than 80.

#### How

[Bonus guide: JoinMarket webui (Jam)](https://raspibolt.org/guide/bonus/bitcoin/Jam.html#bonus-guide-joinmarket-webui-jam) has been updated.

#### Scope

- [ ] significant change to core configuration
- [x] independent bonus guide
- [x] simple bug fix

#### Test & maintenance

Tested with Jam v0.1.5.

### CODE REVIEW

**Comment by dutu** [comment]
Fix for Tor Issue tested and confirmed working by user: 
https://github.com/joinmarket-webui/jam/issues/608#issuecomment-1535343722

File: guide/bonus/bitcoin/Jam.md [hunk 1]
```diff
@@ -171,18 +171,30 @@ $ sudo su - jam
 $ curl https://dergigi.com/PGP.txt | gpg --import
 ```
 
+* Check the latest Jam release version. You can also confirm with the [release page](https://github.com/joinmarket-webui/jam/releases){:target="_blank" rel="noopener"}
+
+  ```sh
+  $ LATEST_VERSION=$(wget -qO- https://api.github.com/repos/joinmarket-webui/jam/releases/latest | grep -oP '"tag_name": "\K(.*)(?=")')
+  $ echo $LATEST_VERSION
+  ```
```
Comment (kristapsk) [inline guide/bonus/bitcoin/Jam.md:179]: Remove spaces at the beginning of lines here, it messes up layout.
```suggestion
```sh
$ LATEST_VERSION=$(wget -qO- https://api.github.com/repos/joinmarket-webui/jam/releases/latest | grep -oP '"tag_name": "\K(.*)(?=")')
$ echo $LATEST_VERSION
```
```

File: guide/bonus/bitcoin/Jam.md [hunk 2]
```diff
@@ -171,18 +171,30 @@ $ sudo su - jam
...
+
 * Retrieve source code
 
+  {: .highlight }
+  > You can also use the latest release version (`$LATEST_VERSION`). However, please be aware that newer releases might not have been thoroughly tested with the rest of the RaspiBolt configuration.
```
Comment (kristapsk) [inline guide/bonus/bitcoin/Jam.md:184]: Same here - spaces at beginning of lines aren't needed.
```suggestion
{: .highlight }
> You can also use the latest release version (`$LATEST_VERSION`). However, please be aware that newer releases might not have been thoroughly tested with the rest of the RaspiBolt configuration.
```

File: guide/bonus/bitcoin/Jam.md [hunk 3]
```diff
@@ -171,18 +171,29 @@ $ sudo su - jam
 $ curl https://dergigi.com/PGP.txt | gpg --import
 ```
 
+* Check the latest Jam release version. You can also confirm with the [release page](https://github.com/joinmarket-webui/jam/releases){:target="_blank" rel="noopener"}
+
+```sh
+$ LATEST_VERSION=$(wget -qO- https://api.github.com/repos/joinmarket-webui/jam/releases/latest | grep -oP '"tag_name": "\K(.*)(?=")')
+$ echo $LATEST_VERSION
```
Comment (kristapsk) [inline guide/bonus/bitcoin/Jam.md:178]: It's now missing closing ```, GitHub code suggestions don't like them.

**Review by kristapsk — APPROVED** [review]
tACK

**Pull request merged**